### DISCUSSION TITLE
Is there a better way to optimize this code?

### DISCUSSION THREAD

**Discussion by facundopadilla** [python-gino/gino#760] [Q&A]
Hi all, I wanted to see recommendations on how to improve this piece of code, I can't think of much because I'm new to using GINO ORM.

My code:
![image](https://user-images.githubusercontent.com/64610246/108447883-77011d80-723f-11eb-896a-73ca54e3ecf6.png)

The result:
![image](https://user-images.githubusercontent.com/64610246/108447852-63ee4d80-723f-11eb-9727-baf264f04eaf.png)

**Comment by wwwjfy** [comment]
This looks good to me already. Anything specific you want to improve?
